docs/misc/localization.md

```diff
@@ -0,0 +1,107 @@
+# Localization
```

Comment:
cc @tsusanka 

Good points! I totally agree that from the docs directory it looks like we are documenting just one thing, but there indeed are many packages and with the addition of connect the current structure will not make much sense anymore.

I'll leave this PR as it is and we can work on the dir structure later.

Reply:
Few thouhgts:
- Clear target where to go when you are a newcomer. `packages/suite/docs/*.md` seems a bit hidden IMHO. Of course we could solve that via summary.
- Is `packages/suite/docs/*.md` better than `docs/packages/suite/*.md`? It is the same IMHO, isn't it?
- I agree that putting everything under `misc` is not a good practice, but honestly I think we should first focus on the writing. The structure will become clear when we have twenty items in `misc` and it becomes ugly. 

----

> It holds several articles that document various parts of the Suite stack. It's not entirely clear if eg. this localization.md applies to just suite project or others as well. Ideally it would be placed roughly at packages/suite/docs/localization.md.

Imho the rule is easy:
- If it concerns just one package, add it to `docs/packages/[package]`.
- If it concerns more/all packages, add it to the root.

---

Discussing it now is imho a pre-mature optimalization. It is easy to reorganize later and it will be easier later when we have more content :).

Reply:
> is packages/suite/docs/*.md better than docs/packages/suite/*.md? It is the same IMHO, isn't it?

I think, yes, practically it is indeed the same. .)

> Discussing it now is imho a pre-mature optimalization. It is easy to reorganize later and it will be easier later when we have more content :).

Right. Let's go this way for some more time. 👍  I'm just wary of writtings 'docs' only to find out later that there's actually lots of them but nobody has the apetite and knowledge to sort them out. .)

Resolving.